Vic2ToHoI4/Source/HOI4World/ShipTypes/PossibleShipVariants.h

```diff
@@ -3,34 +3,30 @@
 
 
 
+#include "LegacyShipVariant.h"
+#include "MtgShipVariant.h"
 #include "newParser.h"
-#include "ShipVariant.h"
 #include <vector>
 
 
 
 namespace HoI4
 {
 
-class possibleShipVariants: commonItems::parser
+class PossibleShipVariants: commonItems::parser
 {
-	public:
-		explicit possibleShipVariants(std::istream& theStream);
+  public:
+	explicit PossibleShipVariants(std::istream& theStream);
 
-		possibleShipVariants() = delete;
-		~possibleShipVariants() = default;
-		possibleShipVariants(const possibleShipVariants&) = default;
-		possibleShipVariants(possibleShipVariants&&) = default;
-		possibleShipVariants& operator=(const possibleShipVariants&) = default;
-		possibleShipVariants& operator=(possibleShipVariants&&) = default;
+	[[nodiscard]] auto getPossibleMtgVariants() const { return mtgVariants; }
+	[[nodiscard]] auto getPossibleLegacyVariants() const { return legacyVariants; }
 
-		[[nodiscard]] auto getPossibleVariants() const { return theVariants; }
-
-	private:
-		std::vector<shipVariant> theVariants;
+  private:
+	std::vector<MtgShipVariant> mtgVariants;
```

Comment:
if you hadn't split legacy and mtg ship variants, you still could have used the same class for both these vectors I think.